src/admin/assignments/views/AssignmentsOverviewAdmin.tsx

```diff
@@ -124,19 +124,19 @@ const AssignmentOverviewAdmin: FunctionComponent<RouteComponentProps & UserProps
 			{
 				id: NULL_FILTER,
 				label: tText('admin/assignments/views/assignments-overview-admin___geen-label'),
-				checked: get(tableState, 'labels', [] as string[]).includes(NULL_FILTER),
+				checked: get(tableState, 'quality_labels', [] as string[]).includes(NULL_FILTER),
```

Comment:
optional: prefer ?. notation to get rid of lodash "get()" function since it has poor type safety